conversions.js

```diff
@@ -0,0 +1,27 @@
+const R = require('ramda');
+
+const toNum = R.curry( (parser,value) => parser(value));
+const orDefault = R.curry((fn,value,data) => fn(data) || value);
+
+const toFloat = toNum(parseFloat);
+const toInt = toNum(parseInt);
```

Comment:
What is the advantage of this over using `parseFloat` or `parseInt` directly? I might be missing something :)